ansible/avena/roles/core/tasks/networking.yml

```diff
@@ -26,7 +26,7 @@
 - name: ensure network manager is enabled
   become: yes
   systemd:
-    name: network-manager
+    name: NetworkManager
```

Comment:
Does this work on a Toradex then? Odd we wouldn't have noticed until now.

Reply:
I asked myself the same when I ran into it. 

It seems both work on the Toradex boards. Strange
```
avena@avena-apalis-dev05:~$ sudo systemctl status NetworkManager
[sudo] password for avena: 
● NetworkManager.service - Network Manager
   Loaded: loaded (/lib/systemd/system/NetworkManager.service; enabled; vendor preset: enabled)
   Active: active (running) since Wed 2020-12-23 19:50:37 UTC; 1 months 9 days ago
     Docs: man:NetworkManager(8)
 Main PID: 393 (NetworkManager)
[...]
avena@avena-apalis-dev05:~$ sudo systemctl status network-manager
● NetworkManager.service - Network Manager
   Loaded: loaded (/lib/systemd/system/NetworkManager.service; enabled; vendor preset: enabled)
   Active: active (running) since Wed 2020-12-23 19:50:37 UTC; 1 months 9 days ago
     Docs: man:NetworkManager(8)
 Main PID: 393 (NetworkManager)
[...]
```

some sort of alias or backwards compatibility?